driver/compile.go

```diff
@@ -134,6 +129,11 @@ func compileMulti(ctx context.Context, program ast.Proc, zctx *resolver.Context,
 	if err != nil {
 		return nil, err
 	}
+	mcfg.Parallelism = len(sources)
+	if mcfg.Parallelism > 1 {
+		program, isParallel = compiler.Parallelize(program, mcfg.Parallelism, sortKey, sortReversed)
+	}
+
```

Comment:
Why move this down here when you could keep it next to the compiler.IsParallelizable call?

Reply:
Oh, it's because you need `len(sources)` for the compiler.Parallelize call. That's hard to see because you're storing it in `mcfg.Parallelism`, so don't.
```suggestion
	if len(sources) > 1 {
		program, _ = compiler.Parallelize(program, len(sources), sortKey, sortReversed)
	}
```